<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_stuManage</class>
 <widget class="QWidget" name="Frame_stuManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <widget class="QWidget" name="stu_add">
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/assets/img/addButton.png</normaloff>:/assets/img/addButton.png</iconset>
    </attribute>
    <attribute name="title">
     <string>添加学生</string>
    </attribute>
    <widget class="QWidget" name="view_add_input" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>201</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btn_addFace">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+ 添加照片</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addstu_id_LineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="tip">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入学号和姓名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addstu_name_LineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="addstu_id_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>学号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="addstu_name_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="stu_update">
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/assets/img/refresh.png</normaloff>:/assets/img/refresh.png</iconset>
    </attribute>
    <attribute name="title">
     <string>更新学生照片</string>
    </attribute>
    <widget class="QWidget" name="view_update_input" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="tb_id_update">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_updateFace">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>更新照片</string>
      </property>
     </widget>
     <widget class="QLabel" name="tip_name_update">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="upstu_id_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>学号:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="upstu_name_Label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="stu_delete">
    <attribute name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/assets/img/finalNode.png</normaloff>:/assets/img/finalNode.png</iconset>
    </attribute>
    <attribute name="title">
     <string>删除学生</string>
    </attribute>
    <widget class="QLabel" name="destu_id_Label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>学号:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_id_delete">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_delete">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="tip_name_delete">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>177</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="destu_name_Label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
